tests/test_arm64.c

```diff
@@ -91,7 +96,7 @@ static void print_insn_detail(cs_insn *ins)
 				break;
 		}
 
-		uint8_t access = op->access;
+		uint32_t access = op->access;
```

Comment:
I thought it caused a warning regarding integer-truncation. But I cannot reproduce it now. It may be my mistake or may be other changes indirectly addressed the warning. I will revert it to uint8_t shortly